src/rendering/SceneManager.cc

```diff
@@ -910,6 +910,19 @@ rendering::MaterialPtr SceneManager::LoadMaterial(
   return material;
 }
 
+void SceneManager::SequenceTrajectories(std::vector<common::TrajectoryInfo>& _trajectories,
+   TP _time)
+{
+  // sequencing all trajectories
+  for (auto &trajectory : _trajectories)
+  {
+    auto duration = trajectory.Duration();
+    trajectory.SetStartTime(_time);
+    _time += dura;
```

Comment:
https://build.osrfoundation.org/job/ignition_gazebo-ci-pr_any-homebrew-amd64/8656/console
```suggestion
    _time += duration;
```